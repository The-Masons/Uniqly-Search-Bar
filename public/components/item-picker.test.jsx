import React from 'react';
import ItemPicker from './item-picker.jsx';

describe('ItemPicker', () => {
  test('should render with the correct items', () => {
    const itemPicker = mount(
      <ItemPicker
        items={[
          {
            name: 'Fake Item 0',
            id: 0,
          },
          {
            name: 'Fake Item 1',
            id: 1,
          },
          {
            name: 'Fake Item 2',
            id: 2,
          },
        ]}/>
      );

    expect(itemPicker).toMatchSnapshot();
  });
});
