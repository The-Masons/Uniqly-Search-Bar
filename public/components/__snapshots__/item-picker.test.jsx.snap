// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemPicker should render with the correct items 1`] = `
<ItemPicker
  items={
    Array [
      Object {
        "id": 0,
        "name": "Fake Item 0",
      },
      Object {
        "id": 1,
        "name": "Fake Item 1",
      },
      Object {
        "id": 2,
        "name": "Fake Item 2",
      },
    ]
  }
>
  <div
    className="itempicker"
  >
    <select
      className="itempicker-select"
    >
      <option
        className="itempicker-option"
        data-id={0}
        key="itemPicker0"
      >
        Fake Item 0
      </option>
      <option
        className="itempicker-option"
        data-id={1}
        key="itemPicker1"
      >
        Fake Item 1
      </option>
      <option
        className="itempicker-option"
        data-id={2}
        key="itemPicker2"
      >
        Fake Item 2
      </option>
    </select>
  </div>
</ItemPicker>
`;
